tests/unit/test_table.py

```diff
@@ -3752,7 +3752,7 @@ def test___repr___explicit(self):
         time_partitioning = self._make_one(
             type_=TimePartitioningType.DAY, field="name", expiration_ms=10000
         )
-        expected = "TimePartitioning(" "expirationMs=10000," "field=name," "type=DAY)"
+        expected = "TimePartitioning(expiration_ms=10000,field=name,type_='DAY')"
```

Comment:
This test still fails, the new expectation of a string value should be quoted now that has been made evaluable:
```suggestion
        expected = "TimePartitioning(expiration_ms=10000,field='name',type_='DAY')"
```